packages/lib/constants.ts

```diff
@@ -104,7 +104,8 @@ export const ENABLE_PROFILE_SWITCHER = process.env.NEXT_PUBLIC_ENABLE_PROFILE_SW
 export const ALLOWED_HOSTNAMES = JSON.parse(`[${process.env.ALLOWED_HOSTNAMES || ""}]`) as string[];
 export const RESERVED_SUBDOMAINS = JSON.parse(`[${process.env.RESERVED_SUBDOMAINS || ""}]`) as string[];
 
-export const ORGANIZATION_MIN_SEATS = 30;
+export const ORGANIZATION_MIN_SEATS = process.env.NEXT_PUBLIC_ORGANIZATIONS_MIN_SELF_SERVE_SEATS || 30;
```

Comment:
I think this could be renamed to `ORGANIZATION_SELF_SERVE_MIN_SEATS` following the convention with the new variable